.xperior/testds/motr-single_tests.yaml

```diff
@@ -20,7 +20,7 @@
 ---
 Tests:
   - id       : 00userspace-tests
-    script   : 'm0 run-ut'
+    script   : 'm0 run-ut -x mdservice-ut'
```

Comment:
It's fine to exclude this if it is a pending bug and will be fixed later.
When it is fixed, we should add it back to the list.